docs/includes/usage-examples/FindManyTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use MongoDB\Laravel\Tests\TestCase;
+
+class FindManyTest extends TestCase
+{
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testFindMany(): void
+    {
+        require_once __DIR__ . '/Movie.php';
+
+        Movie::truncate();
+        Movie::insert([
+            [
+                'title' => 'Centennial',
+                'runtime' => 1256,
+            ],
+            [
+                'title' => 'Baseball',
+                'runtime' => 1140,
+            ],
+        ]);
+
+        // begin-find
+        foreach (
+            Movie::where('runtime', '>', 900)
+            ->orderBy('_id')
+            ->cursor() as $movie
+        ) {
+                echo $movie->toJson() . '<br>';
+        }
```

Comment:
We don't format the code like this usually. That's better to use a variable, and insist on the fact that we get a `Cursor` here.
```suggestion
        $cursor = Movie::where('runtime', '>', 900)
            ->orderBy('_id')
            ->cursor();
        foreach ($cursor as $movie) {
            echo $movie->toJson() . '<br>';
        }
```